pymeasure/display/windows.py

```diff
@@ -465,19 +492,19 @@ def queue(self):
         raise NotImplementedError(
             "Abstract method ManagedWindow.queue not implemented")
 
-    def setup_plot(self, plot):
-        """
-        This method does nothing by default, but can be overridden by the child
-        class in order to set up custom options for the plot
+    # def setup_plot(self, plot):
```

Comment:
I can imagine that people use it to format their graph differently (e.g. set a log-scale). However, I can also imagine that with changing from a single `PlotWidget` to the `widget_list` another implementation is required anyway.

Reply:
If we allow specialization of `TabWidget` then I think the `setup_plot` does not make too much sense. We will break backward compatibility in the interface. In my view the `TabWidget` is something generic, a view of experiment results. It can a plot, it can be a table, it can be anything else, so again the `setup_plot`, if maintained, can be called something like `setup_widget` or similar.